### ISSUE TITLE
The problem with storing data sell and shop

### ISSUE THREAD

**Issue by Pub4Game** [onebone/EconomyS#184]
I used the latest version of the plugin, but I'm having problems with saving data.

**Comment by HmHmmHm** [comment]
What are you using pocketmine version?

**Comment by Pub4Game** [comment]
the latest version of bleeding branches 48

**Comment by onebone** [comment]
I cannot reproduce your error.
Please give me more detailed information.

**Comment by Pub4Game** [comment]
I don't know. I just created a store, restarted the server and it didn't save the stores.

**Comment by CLiu13** [comment]
I am having the same issue. I tried /save-all before a restart, but it doesn't work. If I place about 50 shop/sell signs only about 5-10 save after a restart.

In an attempt to find the source of this problem, I created a shop sign in my default Lobby world and then the rest in my Factions world. The one in the Lobby world seems to always be saved, but most of the signs in the Factions world are un-saved and only a few remain.

Is this because my Faction's world isn't pre-loaded? In my Pocketmine.yml I have set it so that both my Lobby and Factions world will load on startup, which it seems to do when I check the console.

Hopefully this can be fixed soon, I can't really have a factions shop without this!
